app/services/channel.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy.sql import func
+from app.models.channel import Channel
+
+
+async def create_channel(dto, db):
+    new_channel = Channel(
+        application_id=dto.application_id,
+        name=dto.name,
+        # TODO: should discuss Daniel what this Max_members value should be?
+        max_members=20,
```

Comment:
let's set the max members to a static value of 100 for now.

Reply:
ok. I will leave it for Max to set its number